bluebottle/utils/context_processors.py

```diff
@@ -1,8 +1,33 @@
-from django.db import connection
 import re
+import json
 
+from django.db import connection
 from django.conf import settings
 
+from bluebottle.clients import properties
+from tenant_extras.context_processors import tenant_properties as tenant_extra_properties
+
+
+def tenant_properties(request):
+    extra_properties = tenant_extra_properties(request)
+    settings = json.loads(extra_properties['settings'])
+
+    # If tenant has SAML enabled then we also return a list
+    # of read-only user profile properties.
+    try:
+        mappings = properties.TOKEN_AUTH['assertion_mapping']
```

Comment:
Maybe the name should indicate that this is to set the read only properties.

Can't this be decoupled slightly more from tenant_extra_properties a little by this just doing the read only setting?